dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ApplicationConfig.java

```diff
@@ -83,6 +83,9 @@
     // customized parameters
     private Map<String, String> parameters;
 
+    // customized graceful shutdown time
+    private  Integer Wait;

```

Comment:
1. the naming converation should be `wait` or `waitTime`
2. I only see the new field in config, but where do you use this new field?